RedHatInsights/compliance-frontend
# fix(RHINENG-13404): Fix the total number in selected items

This fixes the bulk select showing incorrect number of total selected items when users click on "Select all". The numbers mismatched because of the different fetch arguments being passed to the bulk select hook and to the systems table.

## How to test

1. Go to the /systems page or any other page that shows systems
2. Select all items using the bulk select component
3. Make sure the number of selected items is the same as in pagination

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
